README.md

```diff
@@ -76,7 +76,7 @@ range(1, 200)
 
 For CDN, you can use [unpkg](https://unpkg.com/):
 
-https://unpkg.com/rxjs/bundles/rxjs.umd.min.js
+https://unpkg.com/rxjs/bundles/Rx.min.js
```

Comment:
Oops, this sould actually be: `https://unpkg.com/rxjs@rc/bundles/rxjs.umd.min.js` (because we're in the RC version now)